tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/MsTestV2EvpTests.cs

```diff
@@ -0,0 +1,374 @@
+// <copyright file="MsTestV2EvpTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.Ci;
+using Datadog.Trace.Ci.Tags;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.Ci;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI
+{
+    public class MsTestV2EvpTests : TestHelper
```

Comment:
IMO, it would be preferable to not have separate tests for the alternative protocol. We expect the exact same results in both cases right? So making it a theory test that tests both approaches would be easier for maintenance and avoids divergence IMO

Reply:
Well it's kind of different. the new alternative protocol is not alternative it's the principal protocol (used by agentless and now agent based). This protocol support Test Suite Visibility and model is different from `Span` as APM (for example, there's no Tags dictionary), Also it differ in a couple of tags.

So, yeah we can merge both test but the rewrite to support both (including all the helper methods) it's quite big (kind of duplicate everything) that I think is not worth it.